config/sync/field.storage.node.field_dismissible_option.yml

```
@@ -0,0 +1,30 @@
+uuid: 3b87e7b6-a1d9-4d1b-b509-fba93929532b
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+    - options
+id: node.field_dismissible_option
+field_name: field_dismissible_option
+entity_type: node
+type: list_string
+settings:
+  allowed_values:
+    -
+      value: perm
+      label: 'Don''t allow site visitors to dismiss banner'
+    -
+      value: dismissone
```

```suggestion
      value: dismiss-session
```
dissmissone is hard to interpret meaning from because it could me one time per session, or it could mean dismiss it only once.  It could be interpreted 2 ways.